analyses/cell-type-ewings/scripts/select-cell-types.R

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env Rscript
+
+# This script is used to generate a table of reference cells for a given ScPCA libray that are annotated by 
+# SingleR and/or CellAssign to have a specific label
+# The output is a table that contains columns for barcodes, reference cell class, 
+# and then columns for each of the original cell type annotations for that cell 
+# The `reference_cell_class` column will contain either tumor or normal 
+
+project_root <- here::here()
+renv::load(project_root)
+
+suppressPackageStartupMessages({
+  library(optparse)
+  library(SingleCellExperiment)
+})
+
+option_list <- list(
+  make_option(
+    opt_str = c("--sce_file"),
+    type = "character",
+    help = "Path to RDS file containing a processed SingleCellExperiment object from scpca-nf"
+  ),
+  make_option(
+    opt_str = c("--normal_cells"),
+    type = "character",
+    default = NULL,
```

Comment:
See other comments for why I think this can simplify things a lot!
```suggestion
    default = "",
```